cluster/gce/config-default.sh

```diff
@@ -105,3 +105,6 @@ ENABLE_CLUSTER_DNS=true
 DNS_SERVER_IP="10.0.0.10"
 DNS_DOMAIN="kubernetes.local"
 DNS_REPLICAS=1
+
+# Admission Controllers to invoke prior to persisting objects in cluster
+ADMISSION_CONTROL=NamespaceAutoProvision,LimitRanger,ResourceQuota
```

Comment:
So what happens if these aren't defined? `config-test.sh` doesn't have them (which is what gce e2e uses), and I'm curious what values to use for GKE.